src/docker.rs

```diff
@@ -196,8 +197,10 @@ COPY --chown=solana:solana ./{solana_build_directory}/version.yml /home/solana/
 ENV PATH="/home/solana/.cargo/bin:${{PATH}}"
```

Comment:
Did you mean to revert the changes to not use `.cargo`?

Reply:
what do you mean? I am using `.cargo`. All the built binaries get stored in `/home/solana/.cargo/bin` and then I set the `PATH` here, so you can just run: `solana-bench-tps`, `agave-validator`, etc

Reply:
Oh ok, earlier in the PR you had changed it to go to `/home/solana/bin` instead, and then the last commit went back to using `/home/solana/.cargo/bin`. So is the final decision to use the `.cargo` dir instead?

Reply:
oh ya my bad. not sure exactly what I was thinking and why I switched it back. I must have misread a comment. I am going to switch it back to `/home/solana/bin`